resources.md

```diff
@@ -115,6 +117,18 @@ to get started.
 
 </div>
 
+<div markdown="1" class="item">
+
+#### [Metabase](https://www.metabase.com/)
+
+Metabase is an open-source web based business intelligence platform. You can use Metabase 
+to ask questions about your data, or embed Metabase in your app to let your 
+customers explore their data on their own. More information is available in the [driver 
+project repository](https://github.com/starburstdata/metabase-driver) and the [user 
+guide](https://docs.starburst.io/data-consumer/clients/metabase.html).
```

Comment:
Not sure what the licensing is for the driver, but we should make sure we cover that and make it very clear that this works on Trino and is Apache Licensed for free use. It almost looks like a Starburst ad if we don't include that IMO.

Reply:
Yeah, it's just since the driver is branded with starburst it's very confusing. But yeah hopefully it's clear if it's listed here that it's free.